README.md

```diff
@@ -0,0 +1,61 @@
+# Destiny Scrims Tracker #
+Version 1.0.0
+
+Does you and your clan run a lot of private match scrimmages? Do you have a hard time trying to match up teams to be as fair as possible and still be a good fight?! Well then this is the tool for you! Destiny Scrims Tracker is a web interface to manage all of your internal clan scrimmaging and ranking! With this, all you need to do is input the results of a game and it will calculate a ELO system specific to your clan!
+
+### Models
+
+- [Guardian Object](Destiny.ScrimsTracker.Logic/Models/Guardian.cs)
+- [GuardianMatchResult Object](Destiny.ScrimsTracker.Logic/Models/GuardianMatchResult.cs)
+- [Match Object](Destiny.ScrimsTracker.Logic/Models/Match.cs)
```

Comment:
These don't actually exist yet, right?

Reply:
They do!

Reply:
Ah, just found them. Can they get deleted and re-added via a PR. I have thoughts.

Reply:
Mostly they just there for placeholders. Or did you have thoughts on where they should be?

Reply:
No, on their contents and which models we should have. I have no idea where things should be in a .NET project

Reply:
Done